spec/support/features/pages/base.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Pages
+  class Base
```

Comment:
i feel like you're writing you own page object library. would you consider using something like https://github.com/site-prism/site_prism ?

Reply:
that would indeed be simpler

Reply:
except main has been broken for 5 months

Reply:
done, nice library